example/weather/services/tester/run_tests.go

```diff
@@ -77,6 +77,24 @@ func recoverFromTestPanic(ctx context.Context, testName string, testCollection *
 	}
 }
 
+// Filters a testMap based on a test name that is a glob string
+// using standard wildcards https://tldp.org/LDP/GNU-Linux-Tools-Summary/html/x11655.htm
+func matchTestFilter(ctx context.Context, test string, testMap map[string]func(context.Context, *TestCollection)) (bool, []func(context.Context, *TestCollection)) {
+	match := false
+	testMatches := []func(context.Context, *TestCollection){}
+	var g glob.Glob
+	g = glob.MustCompile(test)
```

Comment:
This will panic if the glob is incorrect which is not acceptable behavior in a service at runtime. Instead the function should return an error that gets surfaced up back to the client of the tester service.

Reply:
Ahh good catch. Will fix.